.code-samples.meilisearch.yaml

```diff
@@ -1061,3 +1061,14 @@ delete_documents_by_filter_1: |-
     --data-binary '{
       "filter": "genres = action OR genres = adventure"
     }'
+get_experimental_features_1: |-
+  curl \
+    -X GET 'http://localhost:7700/experimental-features/' \
+    -H 'Content-Type: application/json'  \
+update_experimental_features_1: |-
+  curl \
+    -X PATCH 'http://localhost:7700/experimental-features/' \
+    -H 'Content-Type: application/json'  \
+    --data-binary '{
+      "scoreDetails": true
+    }'
```

Comment:
FYI: No SDK will support these requests natively.

Reply:
hey @brunoocasali :wave:

Is that a temporary situation, like, on v1.3 release?

I think we had a document about [SDK support for experimental features](https://www.notion.so/meilisearch/Experimental-features-in-SDK-s-1db6388e607f45459ff697d67af3de0f) (internal link), but reading through it I'm unsure what the output of that document is.

At least for the `scoreDetails`, we not stabilizing it immediately because we need feedback on the details of the feature first. Having to do a raw request instead of using the SDK raises the bar for adoption by a lot IMO, so it would be great if some SDK could be updated to have experimental support for some experimental features.

Reply:
Hey!
Since it's a one time activation, it's not a priority for the integration team to have new methods for this.
The contributors will be able to add it, but not something the integration team will work on.

Reply:
🤦 
aaaaaah, sorry I was confused between the activation of the feature (what is being discussed here) and the **use** of the experimental feature (what I had in mind).

OK since it's a time thing.

Do we plan to have the SDK support e.g. the experimental `showRankingScoreDetails` query argument, though?

Reply:
Yes for tier #1 SDK (JS and PHP)
But it will already work on some SDKs (the non-strongly typed languages), like ruby and python